doc/admin-guide/plugins/ts_lua.en.rst

```diff
@@ -131,6 +131,22 @@ is always available within lua script. This package can be introduced into Lua l
 
 `TOP <#ts-lua-plugin>`_
 
+ts.process_id
+-------------
+**syntax:** *val = ts.process_id()*
```

Comment:
hmm. that's not really any field other than "uuid" for "process" at this point. 
I don't have a preference here.